func_server.cpp

```diff
@@ -72,11 +72,9 @@ Status FuncServiceImpl::event(ServerContext* context,
   }
   // potentially non terminating calls
   if (event_type == EVENT::STREAM) {
-    int curr_loop = 0;
-    int loop_max = 5;
     EventReply reply;
     StreamReply streamReply;
-    while (curr_loop != loop_max) {
+    while (true) {
```

Comment:
If user is not registered, it will get caught in the first call to the func layer thus never entering the infinite loop. Good point about the burden, I will sleep the thread between calls